justfile

```diff
@@ -0,0 +1,97 @@
+set windows-shell := ["C:\\Program Files\\Git\\bin\\sh.exe","-c"]
```

Comment:
This is the only make-like magic in the file. Windows is a bit annoying on terminals.